test/integration/default_header_validator_integration_test.cc

```diff
@@ -87,16 +87,9 @@ class DownstreamUhvIntegrationTest : public HttpProtocolIntegrationTest {
   std::string additionallyAllowedCharactersInUrlPath() {
     // All codecs allow the following characters that are outside of RFC "<>[]^`{}\|
     std::string additionally_allowed_characters(R"--("<>[]^`{}\|)--");
-    if (downstream_protocol_ == Http::CodecType::HTTP3) {
-      // In addition H/3 allows TAB and SPACE in path
-      additionally_allowed_characters += +"\t ";
```

Comment:
Don't we still need the HTTP3 case here?

Reply:
I don't believe so; HTTP3 is using the same unified oghttp2 header validator code, and the only thing in that case was adding tabs and spaces, which were dropped.

Reply:
AH, i see. thanks